ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_trackings.schema.yml

```diff
@@ -14,8 +14,8 @@ keys:
       keys:
```

Comment:
Let's create a new key called `flow_tracking` with dictionary items: `sampled` and `hardware`.
We will deprecate flow_tracking and remove it in 5.0.0

Reply:
This would clash with existing `flow_tracking` top level key - maybe you made a typo in this comment?

Reply:
ok deprecated `flow_trackings` - this PR is NOT anymore just adding a type :p 

e1d9085